docs/contributing/test-suite.rst

```diff
@@ -7,10 +7,36 @@
 **********
 Test suite
 **********
+To run the the test suite you'll need nox__, an automated testing tool for Python.
+
+.. code-block:: console
+
+   $ pip install nox
+
+To run the test suite, run this in your checkout of meson-python:
+
+.. code-block:: console
+
+   $ nox -s tests
+
+This will run the test suite for all the python versions you have installed. Some may fail if they aren't a whole installation. e.g. on Ubuntu 22.04, python3.9 will fail unless you've installed it from the relevant PPA. 
```

Comment:
style: could you break these lines? at 80 characters is common in other docs files